apps/frontend/src/app/5_pages/BitocracyPage/components/NewProposalForm/NewProposalForm.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC, useState } from 'react';
+
+import { t } from 'i18next';
+
+import { Select, Button } from '@sovryn/ui';
+
+import { translations } from '../../../../../locales/i18n';
+import {
+  proposalContractOptions,
+  proposalTreasuryOptions,
+  proposalTypeOptions,
+} from './NewProposalForm.constants';
+import { ProposalType } from './NewProposalForm.types';
+
+type NewLoanFormProps = {
+  onSuccess: () => void;
```

Comment:
You don't use this prop at all. Also, I am not sure if this should be `onSuccess` or `onSubmit`, what do you think? Maybe we can provide both and have them optional?

Reply:
This is something we can add when when needed. so i'll remove it for now